### ISSUE TITLE
python: IDLE crashes when dragging files onto dock icon

### ISSUE THREAD

**Issue by agentirons** [Homebrew/homebrew-core#12068]
# Please always follow these steps:
- [X] Confirmed this is a problem with only one, specific formula and not Homebrew/brew? If it's a general Homebrew/brew problem please file this issue at https://github.com/Homebrew/brew/issues/new
- [X] Ran `brew update` and retried your prior step?
- [X] Ran `brew doctor`, fixed as many issues as possible and retried your prior step?
- [X] Ran `brew gist-logs <formula>` (where `<formula>` is the name of the formula that failed) and included the output link?
https://gist.github.com/ec46d0b33a1cc6ccf07d968c367c2ce7
- [ ] If `brew gist-logs` didn't work: ran `brew config` and `brew doctor` and included their output with your issue?

To help us debug your issue please explain:
- What you were trying to do (and why)
When IDLE is already open, I would like to open a .py file by dragging it onto IDLE's dock icon.

- What happened (include command output)
IDLE crashes immediately. I can open as many files as I want using the File->Open... menu command, and if IDLE is not already open then I can open a single file by dragging it onto my dock icon, but any more files dragged on will crash IDLE. I am running OSX 10.11.6, and have reproduced this behavior on two different computers (one Hackintosh, one real). The same problem occurs with python3 and IDLE3.

I can confirm that after removing all brew packages, installing ActiveTcl8.5.18.0, and installing Python 2.7.13 from the official OS X installer package, this issue is not present - I can drag as many files onto the IDLE dock icon as I like, no trouble.

- What you expected to happen
I expected to be able to open any .py files by dragging them onto IDLE's dock icon.

- Step-by-step reproduction instructions (by running `brew install` commands)
(Starting from a completely fresh installation of brew)
$ brew install tcl-tk
$ brew install python --with-tcl-tk
$ idle
Attempt to drag a file onto IDLE's dock icon.

**Please note we may immediately close your issue without comment if you do not fill out the issue template and provide ALL the requested information.**

# New Formulae
To get a new formula into Homebrew please file a [Pull Request](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md).

**Comment by dunn** [comment]
cc @tdsmith

**Comment by tdsmith** [comment]
Thanks for the message; sorry you're having trouble. Can you get a traceback or a crash dump? I don't see an obvious way to troubleshoot this without knowing way more about IDLE; it's possible you could get a more helpful response on [python-list](https://mail.python.org/mailman/listinfo/python-list).

**Comment by agentirons** [comment]
Sorry for the delay, I had to get another computer up and running with this setup to get a crashlog, but it is attached.

I also noticed in the process that python installed by itself through homebrew, using Apple's tcl-tk, doesn't exhibit this behavior. It is in fact only a problem when the brewed python and tcl-tk are used together.

[idle_crashlog.txt](https://github.com/Homebrew/homebrew-core/files/920285/idle_crashlog.txt)
